app/coffeescript/components/mixins/stored_markers.coffee

```diff
@@ -12,25 +12,26 @@ define [
 
     @defaultAttrs
       storageKey: 'viewedMapMarkers'
+      store: undefined
```

Comment:
Should `store` really be a FlightJS attribute?
Seems like it could just be a plain old js attribute. (so in coffee, `@store` instead of `@attr.store`).
Let me know if you'd like to change that in this PR or not... I'm happy to go ahead and merge.
There are lots of other places in map.js (and probably other Flight code) where we misuse Flight attributes to store mutable state. Pretty sure they are intended for immutable configuration options. 
I've misused them myself copying the prevailing style.
Maybe we need a tech debt chore to clean up this issue in map.js.